overrides.rb

```diff
@@ -75,9 +75,11 @@
     end
 
     if pkg.kind_of?(Autobuild::Ruby)
-        if pkg.test_utility.enabled? && pkg.respond_to?(:rake_test_options) # autoproj v2 only
-            pkg.depends_on 'minitest-junit'
-            pkg.rake_test_options << "TESTOPTS=--junit --junit-filename=#{pkg.test_utility.source_dir}/report.junit.xml --junit-jenkins"
+        pkg.post_import do
```

Comment:
post import hooks are called even when don't do an import operation, right?

Reply:
Yes. From the perspective of Autoproj 'import' is 'possibly import and load configuration'